Makefile

```diff
@@ -22,6 +22,9 @@ install:  ## Build and install binaries in $GOBIN or $GOPATH/bin
 run: build ## Run foxtrot server
 	$(O)/foxtrot
 
+pkg/foxtrot/schema.go: sql/schema.sql
+	@printf '//generated DO NOT EDIT\n\npackage foxtrot\n\nconst schema = `%s`\n' "$$(cat $<)" > $@
```

Comment:
note this only works if `sql/schema.sql` has no backticks in it. but i think this is good enough until we start embedding it with the new go embedding.